src/presenters/gouvernancePresenter.ts

```diff
@@ -4,10 +4,11 @@ import { isNullish } from '@/shared/lang'
 import { ComiteReadModel, FeuilleDeRouteReadModel, MembreReadModel, UneGouvernanceReadModel } from '@/use-cases/queries/RecupererUneGouvernance'
 
 export function gouvernancePresenter(
-  gouvernanceReadModel: UneGouvernanceReadModel
+  gouvernanceReadModel: UneGouvernanceReadModel,
+  now = (): Date => new Date()
```

Comment:
```suggestion
  now: Date
```
En mettant une valeur par défaut, tu n'as pas vu tout de suite dans tes tests qu'il fallait mettre ce paramètre plutôt qu'un spy.

Reply:
Ne faut-il pas que ça reste une fonction retournant une date ?